### DISCUSSION TITLE
Wiki

### DISCUSSION THREAD

**Discussion by ikanakova** [fast4x/RiMusic#58] [Ideas]
I don't know how to create a Wiki on Github, so I've created a [Wiki.md](https://github.com/ikanakova/RiMusic/blob/master/WIKI.md) file with instructions on how to translate the app for those who aren't very good with Github. @fast4x should I open a pull request or will you create a Wiki page from my Wiki.md?

**Comment by ikanakova** [comment]
Additionally, the Wiki could explain how to set up the theme, gestures in the app, etc.

**Comment by fast4x** [comment]
I check how to create wiki on Github. Hopefully, it can be done on the RiMusic website if it can't be done on Github.

**Comment by fast4x** [comment]
Wiki now Is enabled

**Reply by ikanakova** [reply]
Great!
Are you going to add the [translation page](https://github.com/ikanakova/RiMusic/blob/master/WIKI.md) or are you going to [allow everyone to contribute to the Wiki](https://docs.github.com/en/communities/documenting-your-project-with-wikis/changing-access-permissions-for-wikis) so I can put it there?

**Reply by fast4x** [reply]
By settings, all contributors are enabled to add/edit wiki pages.

**Reply by fast4x** [reply]
Try now please

**Reply by ikanakova** [reply]
It can be done now😊 I added [Contributor's Guide: Adding Translations]( https://github.com/fast4x/RiMusic/wiki/Contributor's-Guide:-Adding-Translations).

**Reply by ikanakova** [reply]
I'm hoping it'll work that way without the fork. I think making a fork is much more complicated.

**Reply by fast4x** [reply]
You could differentiate, 

- first method > fork....

- second method > open a feature request or ticket attaching the translation file

**Reply by ikanakova** [reply]
Done, Wiki modified. I hope it's OK like this.

**Reply by fast4x** [reply]
Great!

**Comment by twistios** [comment]
I created a new wiki-page [How to use the app](https://github.com/fast4x/RiMusic/wiki/How-to-use-the-app).
I was inspired to do this because people asked how one can change the sorting of artists (like #2044, #2242 & #2555)

**Reply by fast4x** [reply]
Ok thanks, manage it.

**Comment by twistios** [comment]
I added a new [section](https://github.com/fast4x/RiMusic/wiki/FAQs#how-do-i-get-log-files-in-case-of-a-crash) in FAQ about crash logs.
